packages/stateful/proposal-module-adapter/adapters/CwdProposalMultiple/components/ProposalLine/ProposalStatus.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Block,
+  CheckCircle,
+  CheckCircleOutline,
+  RemoveCircle,
+  StopCircleOutlined,
+  Timelapse,
+} from '@mui/icons-material'
+import clsx from 'clsx'
+import { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { StatusDisplay } from '@dao-dao/stateless'
+import { Status } from '@dao-dao/types/contracts/CwdProposalMultiple'
+
+export interface ProposalStatusProps {
+  status: Status
+  // Dim
+  dimmed?: boolean
+}
+
+export const ProposalStatus = ({
```

Comment:
since this is the same as the single choice proposal (assuming the status type is the same), we can probably move this to the stateless package and use the same one everywhere